lib/tasks/csv_report.rake

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "csv"
+
+namespace :csv_report do
+  desc "A CSV report of political documents published by organisations"
+  task political_status_by_organisation: :environment do
+    raise "csv_report:political_status_by_organisation rake task should be removed" unless Time.zone.today < Date.new(2020, 1, 1)
+
+    # Get all primary publishing organisations in use
+    org_content_ids = Edition.where(live: true).joins(revision: :tags_revision).pluck(Arel.sql("tags_revisions.tags->'primary_publishing_organisation'->0")).uniq
```

Comment:
This line would be easier to read if it was split over several lines 😄 Haven't come across Arel before, nice!

Reply:
Makes sense to split it.
I wasn't initially going to use Arel but this warning popped up:
```
DEPRECATION WARNING: Dangerous query method (method whose arguments are used
as raw SQL) called with non-attribute argument(s): "tags_revisions.tags
->'primary_publishing_organisation'->0". Non-attribute arguments will be 
disallowed in Rails 6.0. This method should not be called with user-provided
values, such as request parameters or model attributes. Known-safe values can 
be passed by wrapping them in Arel.sql().
```